src/templates/react/Content/FableElmishReactTemplate.fsproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="src/Common.fs" />
+    <Compile Include="src/Navbar.fs" />
+    <Compile Include="src/Pages/About.fs" />
```

Comment:
This is highly subjective of course, but personally something [like this](http://blog.jenkster.com/2016/04/how-i-structure-elm-apps.html) is the file organization I encourage.